akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/Effect.scala

```diff
@@ -58,6 +63,18 @@ object Effect {
    * Side effects can be chained with `andThen`
    */
   def stop[Event, State]: Effect[Event, State] = none.andThenStop()
+
+  // FIXME doc all reply and thenReply stuff
+
+  def reply[ReplyMessage, Event, State](cmd: ExpectingReply[ReplyMessage])(replyWithMessage: ReplyMessage): ReplyEffect[Event, State] =
+    none[Event, State].thenReply[ReplyMessage](cmd)(_ ⇒ replyWithMessage)
+
+  def replyDone[Event, State](cmd: ExpectingReply[Done]): ReplyEffect[Event, State] =
+    reply(cmd)(Done)
```

Comment:
Now the user may be in the other, default mode, and accidentally use these and think there will be an automagic reply. So we have gone from having the risk of missing out on sending a reply, to having the risk of thinking that you replied but did in fact not?

Reply:
nope, there are no automagic replies here, these effects will also work when you use them in a non-enforcing PersistentBehavior

Reply:
Ah, good, it's just that they aren't enforced. Got it 👍